uiowa/uiowa
# [Facilities] Artist person type, artist artwork view, buliding artwork view, and artwork search block

Resolves https://github.com/uiowa/uiowa/issues/7113
Resolves https://github.com/uiowa/uiowa/issues/7144
Resolves https://github.com/uiowa/uiowa/issues/7150

# How to test

```
ddev blt ds --site=facilities.uiowa.edu  && ddev drush @facilities.local uli /node/3636/edit
```


- Edit /node/3636/edit and select `Artist` and confirm that new fields appear.   Save and confirm that new fields appear above the biography. 
- Confirm "Artwork" block appears on the person node page with teaser cards in a three column grid.
- Go to https://facilities.uiowa.ddev.site/building/0189 and confirm that artwork teasers are displayed on only that page. 
- Add a new "Page" and confirm that the "Artwork: Search block" Shows up in "Site Custom".   Add the block and test out the search functionality.
